js/apps/system/_admin/aardvark/APP/frontend/js/views/documentView.js

```diff
@@ -453,21 +453,23 @@
       model = JSON.stringify(model);
 
       if (this.type === 'edge' || this.type._from) {
-        var callbackE = function (error, data, navigate) {
+        var callbackE = function (error, data) {
           if (error) {
             arangoHelper.arangoError('Error', data.responseJSON.errorMessage);
           } else {
-            // here we need to do an additional error check as PUT API is returning 202 (PUT) with error inside - lol
-            if (data[0].error) {
-              arangoHelper.arangoError('Error', data[0].errorMessage);
+            // Here we need to do an additional error check as PUT API is returning 202 (PUT) with error inside.
+            // Can contain error variable, but can contain errorNum and errorMessage as well (Babies API)
+            const entry = data[0];
+            if (entry.error || (entry.errorNum && entry.errorMessage)) {
```

Comment:
Sorry I missed this bit in the main PR, but noticed in the backport. It seems `entry.errorNum` and `entry.errorMessage` are assumed to exist if `entry.error` exists. Is this a correct assumption? If so, we can simply omit the check for `entry.error`.